File: ReceiptRewards.PCL/Models/FormFlow.cs
using System.Collections.Generic;

namespace ReceiptRewards.PCL.Models {

    public class FormFlow {

        private int _formFlowId;
        private int _formId;
        private Question _question;
        private string _value;


        public int formFlowId {
            get { return _formFlowId; }
            set { _formFlowId = value; }
        }
        
        public int formId {
            get { return _formId; }
            set { _formId = value; }
        }

        public Question question {
            get { return _question; }
            set { _question = value; }
        }

        public string value {
            get { return _value; }
            set { _value = value; }
        }

    }
}

File: ReceiptRewards.WebServices/src/com/model/FormFlow.java
package com.model;

import javax.xml.bind.annotation.XmlRootElement;

@XmlRootElement
public class FormFlow {

    private int formFlowId;
    private int formId;
    private Question question;
    private String value;

    public FormFlow() {
        this.formFlowId = 0;
        this.formId = 0;
        this.question = null;
        this.value = "";
    }
    
    public FormFlow(int formFlowId, int formId, Question question, String value) {
        this();
        this.formFlowId = formFlowId;
        this.formId = formId;
        this.question = question;
        this.value = value;
    }

    public int getFormFlowId() {
        return formFlowId;
    }

    public void setFormFlowId(int formFlowId) {
        this.formFlowId = formFlowId;
    }

    public int getFormId() {
        return formId;
    }

    public void setFormId(int formId) {
        this.formId = formId;
    }

    public Question getQuestion() {
        return question;
    }

    public void setQuestion(Question question) {
        this.question = question;
    }

    public String getValue() {
        return value;
    }

    public void setValue(String value) {
        this.value = value;
    }
    
    @Override
    public String toString() {
    	// TODO: Implement
        return "";
    }
}
